Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrElementBuilders.kt

```diff
@@ -466,6 +470,8 @@ internal class FunctionStubBuilder(
             context.mirror(func.returnType).argType
         }.toStubIrType()
 
+        if (skipOverloads && isOverloading(func.name))
```

Comment:
Do we guarantee that two consequent `cinterop` invocations will return the same result?

Reply:
@sbogolepov  Don't catch the question: nothing is shared between  `cinterop` invocations.
Do you suggest to move `skipOverloads` to `buildingContext` or `InteropConfiguration`?

Reply:
I'm talking about an order of incoming declarations. Is it possible that on the first invocation of `cinterop` the input is `[A, B, C]` and on the second one it is `[B, C, A]`?

Reply:
> Do you suggest to move skipOverloads to buildingContext or InteropConfiguration?
I think it's fine where it is.